packages/mobile/src/home/CashInBottomSheet.test.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import { fireEvent, render } from 'react-native-testing-library'
+import { Provider } from 'react-redux'
+import * as renderer from 'react-test-renderer'
+import CashInBottomSheet from 'src/home/CashInBottomSheet'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { createMockStore } from 'test/utils'
+
+describe('CashInBottomSheet', () => {
+  it('renders correctly', () => {
+    const tree = renderer.create(
+      <Provider store={createMockStore({})}>
+        <CashInBottomSheet />
+      </Provider>
+    )
+    expect(tree).toMatchSnapshot()
+  })
+
+  it('it navigates to the add funds page when the add funds button is clicked', () => {
```

Comment:
Nit: you don't need to start the sentence with "it" as it's implied by the testing framework